Source/spells.cpp

```diff
@@ -212,45 +212,44 @@ void CastSpell(int id, int spl, int sx, int sy, int dx, int dy, int spllvl)
 
 static void PlacePlayer(int pnum)
 {
-	int nx, ny, max, min, x, y;
+	int max, min, x, y;
 	DWORD i;
 	bool done;
+	Point newPosition = {};
 
 	if (plr[pnum].plrlevel == currlevel) {
 		for (i = 0; i < 8; i++) {
-			nx = plr[pnum].position.tile.x + plrxoff2[i];
-			ny = plr[pnum].position.tile.y + plryoff2[i];
-
-			if (PosOkPlayer(pnum, nx, ny)) {
+			newPosition = plr[pnum].position.tile + Point { plrxoff2[i], plryoff2[i] };

```

Comment:
Was merging `plrxoff2` + `plryoff2` in to a single `Point` array already mentioned?

Reply:
Obligaron mentioned it. We'll get there.